src/meetings/components/meetingAccessPoint/externalAccess/MeetingExternalAccessPage.test.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React from 'react';
+
+import { screen } from '@testing-library/react';
+
+import MeetingExternalAccessPage from './MeetingExternalAccessPage';
+import { MeetingsApiToSpy, spyOnMeetingsApi } from '../../../../tests/mocks/network';
+import { setup } from '../../../../tests/test-utils';
+
+describe('MeetingExternalAccessPage tests', () => {
+	test('Meeting name is displayed correctly', async () => {
+		spyOnMeetingsApi(MeetingsApiToSpy.GET_SCHEDULED_MEETING_NAME).mockResolvedValue({
+			name: 'Test Meeting'
+		});
+		setup(<MeetingExternalAccessPage />);
+
+		expect(await screen.findByText('Welcome to "Test Meeting" virtual room')).toBeInTheDocument();
+	});
+
+	test('Create new guest', async () => {
+		const spyCreateGuest = spyOnMeetingsApi(
+			MeetingsApiToSpy.CREATE_GUEST_ACCOUNT
+		).mockResolvedValueOnce({});
+
+		const { user } = setup(<MeetingExternalAccessPage />);
+		const nameInput = await screen.findByPlaceholderText('Enter your name');
+		await user.type(nameInput, 'Guest User');
+		const readyButton = await screen.findByText('Ready to participate');
+		await user.click(readyButton);
+		expect(spyCreateGuest).toHaveBeenCalled();
+	});
+
+	test('Redirect to login page', async () => {
+		window.location.pathname = 'https://localhost/carbonio/meetings/meeting-id';
+		const { user } = setup(<MeetingExternalAccessPage />);
+		const loginButton = await screen.findByText('Go to your login page');
+		await user.click(loginButton);
+		expect(window.location.replace).toBeCalled();
```

Comment:
it's better to use `toHaveBeenCalled()` since `toBeCalled()` is deprecated
```suggestion
		expect(window.location.replace).toHaveBeenCalled();
```